### PR TITLE
AC-538 Add Snackbar when going online

### PR DESCRIPTION
**Pull request by Abdelaty**

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java [hunk 1]
```diff
@@ -161,10 +162,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     Intent intent = new Intent("org.openmrs.mobile.intent.action.SYNC_PATIENTS");
                     getApplicationContext().sendBroadcast(intent);
                     ToastUtil.showShortToast(getApplicationContext(), ToastUtil.ToastType.NOTICE, R.string.reconn_server);
-                    if (PatientDetailsFragment.snackbar != null)
+                    if (PatientDetailsFragment.snackbar != null) {
                         PatientDetailsFragment.snackbar.dismiss();
-                    if (snackbar != null)
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java]: @Abdelaty Don't remove this - this is a functionality of the `no internet connection` snackbar. Just create another if statement below it.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java [hunk 2]
```diff
@@ -204,6 +209,16 @@ public void onNext(List<Location> locations) {
         };
     }
 
+
+    private void showInternetConnectionSnackbar() {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java]: @Abdelaty Can you change the function name to `showConnectedToInternetSnackbar`

Comment (Abdelaty) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java]: Sure

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java (hunk 1 above)
Comment (Abdelaty) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/ACBaseActivity.java]: In line 169 I have added it again.

**Pull request closed without merging**